app/views/userInterface/index.ejs

```diff
@@ -1,56 +1,64 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <title><%= title %></title>
-        <meta charset="utf-8">
-        <meta http-equiv="Content-Security-Policy" content="script-src 'self' 'sha256-WOdSzz11/3cpqOdrm89LBL2UPwEU9EhbDtMy2OciEhs='">
-        <script src="javascripts/jquery/jquery.min.js"></script>
-        <script src="javascripts/datatable/datatable.min.js"></script>
-        <script src="javascripts/uiheader.js"></script>
-        <!-- Compiled and minified CSS -->
-        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
-        <!-- Compiled and minified JavaScript -->
-        <script src="javascripts/materializejs/materialize.min.js"></script>
 
-        <link href="stylesheet/datatable/datatable.min.css" rel="stylesheet">
-        <link href="stylesheet/dashboard.css" rel="stylesheet" type="text/css">
-    </head>
-    <body>
-        <header>
-            <%- include('../components/banner.ejs') %>
-        </header>
-        <main>
-            <%- include('../components/headers.ejs') %>
-            <div class="valign-wrapper table-box">
-            <div class="row  table-box">
-                <div class="col card s10 push-s1">
-                    <div class="row">
-                        <table id="datatable">
-                            <thead>
-                                <tr>
-                                    <th>Class ID</th>
-                                    <th>Created At</th>
-                                    <th>Updated At</th>
-                                    <th>Scheduled Time</th>
-                                    <th>Status</th>
-                                </tr>
-                            </thead>
-                            <tbody>
-                            <%jobs.map(job => {%>
+<head>
```

Comment:
One last thing, this file is a little bit out dated, that's why it created all these differences, you can find the new version of this file in the main branch, and just copy and paste it here to push again so that these diffs can be gone.